src/app/task-form/task-form.component.html

```diff
@@ -0,0 +1,15 @@
+<form [formGroup]="taskForm" (ngSubmit)="onSubmit()">
+
+	<span class="p-float-label" style="margin-bottom:15px">
+		<input type="text" id="title" formControlName="title" placeholder="Task Titl" pInputText>
+		<label *ngIf="!taskForm.value.title" for="inputtext">Task Title</label>
```

Comment:
```suggestion
		<input type="text" id="title" formControlName="title" pInputText>
		<label for="title">Task Title</label>
```
if you have floating label, you do not need placeholder anymore

Reply:
Before
![image](https://user-images.githubusercontent.com/2982269/186379307-05bfa649-2173-4da7-8506-628b68ab7852.png)


After
![image](https://user-images.githubusercontent.com/2982269/186379193-2b0273f7-1a7f-45fb-9596-3576371a5aa3.png)

I think that ngIf should be left there

Reply:
check the docs https://www.primefaces.org/primeng/inputtext

Reply:
see also the source tab from the link